docs/src/app/(public)/(content)/react/utils/use-render/demos/render-callback/css-modules/index.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import styles from './index.module.css';
+
+type CounterState = {
+  odd: boolean;
+};
+
+type CounterProps = {
+  className?: string;
+  render?: useRender.RenderProp<CounterState>;
+};
+
+function Counter(props: CounterProps) {
+  const { render, className, ...otherProps } = props;
+  const [count, setCount] = React.useState(0);
+  const odd = count % 2 === 1;
+  const state = React.useMemo(() => ({ odd }), [odd]);
+
+  const { renderElement } = useRender({
+    render: render ?? <button type="button" />,
```

Comment:
We should `<button type="button">` as a default assignment as we do in the other demo.

Reply:
What do you mean?

Reply:
Move it to `const { render = <button type="button" />, className, ...otherProps } = props;`